collectors/python.d.plugin/openldap/openldap.chart.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# Description: openldap netdata python.d module
+# Author: Manolis Kartsonakis (ekartsonakis)
+# SPDX-License-Identifier: GPL-3.0+
+
+try: 
+     import ldap 
+     HAS_LDAP = True 
+except ImportError: 
+     HAS_LDAP = False
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+# default module values (can be overridden per job in `config`)
+update_every = 10
+priority = 60000
+retries = 60
+
+ORDER = [
```

Comment:
please fromat it like this
https://github.com/netdata/netdata/blob/f8e0a57b2e6a77d4b0aac38c864cd02c66cd998b/collectors/python.d.plugin/adaptec_raid/adaptec_raid.chart.py#L19-L24

Reply:
done